pybb/templates/pybb/add_post.html

```diff
@@ -1,5 +1,5 @@
 {% extends 'pybb/base.html' %}
-{% load i18n pybb_tags staticfiles %}
+{% load i18n pybb_tags static %}
```

Comment:
This commit follows the following line
> The staticfiles and admin_static template tag libraries are removed.

from: https://docs.djangoproject.com/en/dev/releases/3.0/#features-removed-in-3-0